code-challenges.test.js

```diff
@@ -14,38 +14,166 @@
 
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
+  // Pseudocode:
+    // Given 2 arrays
+    // Create describe function named "shuffleWithoutFirstItem"
+    // The first test will check to see if the function correctly shuffles the elements in the input but leaving out the first element
+    // The second test will check to make sure the new array returns the remaining colors in a random order and if it contains the items from the expected output
+    // Use 'expect.arrayContaining to check if the colors are the same as the expected output
+    // 
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = [
-  "chartreuse",
-  "indigo",
-  "periwinkle",
-  "ochre",
-  "aquamarine",
-  "saffron"
-]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+
+    describe("shuffleWithoutFirstItem", () => {
+      it("shuffles an array without its first item", () => {
+        const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+        const shuffledColors1 = shuffleWithoutFirstItem(colors1)
+        expect(shuffledColors1).toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]))
+    
+        const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+        const shuffledColors2 = shuffleWithoutFirstItem(colors2)
+        expect(shuffledColors2).toEqual(expect.arrayContaining(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]))
+      })
+      }) // output:  FAIL  ./jest.test.js
+      //   ● shuffleWithoutFirstItem › shuffles an array without its first item
+      
+      //     ReferenceError: shuffleWithoutFirstItem is not defined
+      
+      //       60 |     it("shuffles an array without its first item", () => {
+      //       61 |       const colors1 = ["purple", "blue", "green", "yellow", "pink"];
+      //     > 62 |       const shuffledColors1 = shuffleWithoutFirstItem(colors1);
+      //          |                               ^
+      //       63 |       expect(shuffledColors1).toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]))
+      //       64 |   
+      //       65 |       const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 
 // b) Create the function that makes the test pass.
+      // Got stuck on the function
+const shuffleWithoutFirstItem = (array) => {
+  const firstItem = array.shift();
+  const shuffled = array.sort(() => Math.random() - ???);
+  // 
+  return shuffled;
+}
+
```

Comment:
So close! The thought process is great, just missing that last piece. Think about what Math.random() is giving you, as well as what is happening within the `.sort()` method. You are right at the edge here! Also, be sure to include your pseudocode, even if you got stuck at that last part! If you have further questions, don't hesitate to message me on Slack!

Reply:
Ughhh I knew I messed this one up. Thank you for giving me insight on what to do next!